Resources/Locale/en-US/objectives/conditions/doorjack-condition.ftl

```diff
@@ -0,0 +1,2 @@
+objective-condition-doorjack-title = Doorjack {$count} doors on the station.
+objective-condition-doorjack-description = Use your gloves to doorjack {$count} airlocks on the station.
```

Comment:
Did you mean to say "Airlocks" here? Most players might not realize that all doors are airlocks.

Reply:
well you cant emag a wood or steel door